projects/make-a-vis/src/lib/mav-selection/graphic-variable-type/graphic-variable-type.component.ts

```diff
@@ -77,13 +92,19 @@ export class GraphicVariableTypeComponent implements OnChanges {
   constructor(
     private store: Store<any>,
     private updateService: UpdateVisService,
-    private hoverService: DataVariableHoverService
+    private hoverService: DataVariableHoverService,
+    private advancedService: AdvancedService,
+    private mavSelectionStore: Store<MavSelectionState>
   ) {
-    store.pipe(select(getAvailableGraphicVariablesSelector)).subscribe((availableGraphicVariables) => {
+    this.advancedService.addAdvancedCheat();
```

Comment:
Can't advancedService just call addAdvancedCheat in its constructor? Seems unnecessary for this detail to be anywhere else.